documentation/docs/pages/developers/typescript/api/sdk/globals.md

```
@@ -1,33 +1,56 @@
 
-- [NymMixnetClientOptions](interfaces/NymMixnetClientOptions.md)
-- [NymMixnetClient](interfaces/NymMixnetClient.md)
-- [Client](interfaces/Client.md)
-- [Events](interfaces/Events.md)
-- [LoadedEvent](interfaces/LoadedEvent.md)
-- [ConnectedEvent](interfaces/ConnectedEvent.md)
-- [StringMessageReceivedEvent](interfaces/StringMessageReceivedEvent.md)
-- [BinaryMessageReceivedEvent](interfaces/BinaryMessageReceivedEvent.md)
-- [RawMessageReceivedEvent](interfaces/RawMessageReceivedEvent.md)
-- [Payload](interfaces/Payload.md)
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)